bin/build_daq

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+image_name="daq"
+
+if [ "$1" == "dts" ]; then
+  image_name="dts"
+  shift
+fi
+
+bin/setup_remote faucet
```

Comment:
Script is misnamed since it's not "building daq" as such... so maybe "build_daq_container"?  (dts was specific to it being a container, daq is not so qualified).

Reply:
Makes sense. Canged.